SDLTooSharp/Managed/Video/Surface/ISurface.cs

```diff
@@ -1,17 +1,106 @@
+using SDLTooSharp.Bindings.SDL2;
 using SDLTooSharp.Managed.Common;
+using SDLTooSharp.Managed.Exception.Video.Surface;
 
 namespace SDLTooSharp.Managed.Video.Surface;
+
 /// <summary>
 /// Interface for surfaces
 /// </summary>
-public interface ISurface
+public interface ISurface : IDisposable
 {
```

Comment:
Have you considered documenting the initialization behavior of the `SurfacePtr` property? This can be crucial for users working with the interface to understand proper usage.